@ndlib/gatsby-theme-marble/src/assets/theme.js

```diff
@@ -126,7 +126,7 @@ export const theme = {
       },
     },
     a: {
-      color: 'primary',
+      color: '#FFFFFF',
```

Comment:
This is the default link color for and for most content that is assumed to be on `'background'` which is also white. This should remain a dark color. Links that should be white because they are on a dark background should use `color: 'background',` and be set in their local `sx` prop or` sx.js` file.

Reply:
Yup, you're absolutely right. I updated the wrong link color value, but as it turns out, an update to the footer color value doesn't appear to be needed after all (i.e. `Footer a: color: 'primary'`. I think something goofy was going on with my cache.